### DISCUSSION TITLE
[BUG] When left sidebar is closed and page reloaded, it is initially rendered as open

### DISCUSSION THREAD

**Discussion by stodge** [ColorlibHQ/AdminLTE#3319] [General]
**Describe the bug**
Maybe I don't understand the intent of this, but I modified the left sidebar to remember it's state after a reload- open or closed:

```
        <a class="nav-link" data-widget="pushmenu" data-enable-remember="TRUE" data-no-transition-after-reload="TRUE" href="#" role="button"><i class="fas fa-bars"></i></a>
```

When I reload the page, the left sidebar is rendered open, and then automatically closes, which doesn't look too good.

I tried  both TRUE and FALSE for no-transition-after-reload.

**To Reproduce**
1. Add the two data items to the sidebar as per the code fragment above 
2. Reload and the click the burger icon to close the sidebar
3. Reload the page
4. The sidebar is immediately rendered open and then automatically closes

**Expected behavior**
I would expect the sidebar to be initially rendered as closed after a reload. I wouldn't expect it to be rendered open first and then closed.

- AdminLTE Version: v3.0.5
- Operating System: Debian (ChromeOS Crostini)
- Browser (Version): Chrome Version 87.0.4280.109 (Official Build) (64-bit)

Thanks

**Comment by pedropamn** [comment]
https://adminlte.io/docs/3.2/layout.html ("Preloader" section)
